common/djangoapps/django_sudo_helpers/tests/test_startup.py

```diff
@@ -0,0 +1,30 @@
+"""
+Tests django-sudo startup
+"""
+from django.conf import settings
+from django.test import TestCase
+from mock import patch
+from django_sudo_helpers.startup import run
+
+
+class DjangoSudoStartupTestCase(TestCase):
+    """
+    Test django-sudo startup
+    """
+
+    def setUp(self):
+        super(DjangoSudoStartupTestCase, self).setUp()
+
+    @patch.dict("django.conf.settings.FEATURES", {"ENABLE_THIRD_PARTY_AUTH": False})
```

Comment:
@dan-f , I think that's normally the right thing to do, but for `FEATURES`, we don't want to override the whole dictionary, we just want to override the one key-value pair in `FEATURES`.